addons/sale_purchase/tests/test_sale_purchase.py

```diff
@@ -364,3 +364,36 @@ def test_service_to_purchase_multi_company(self):
         # FIXME: same sudo issue as above
         order2.sudo().with_company(company_2).action_confirm()
         self.assertTrue(order2.purchase_order_count)
+
+    def test_service_to_purchase_branch_tax_propagation(self):
+        """
+            Ensure that SO/PO of a branch can use root company's taxes
```

Comment:
```suggestion
        Ensure that SO/PO of a branch can use root company's taxes
```
Yes, this is nitpicking héhé